1008 Theming the different blocks on the homepage

Javi-er opened the pull request:
Issue #1008 

This is a work in progress

Javi-er committed:
Theming the homepage

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
changing visibility settings for page title

Javi-er committed:
adjusting display settings

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
Merge branch 'main' into 1008--landing-page-and-blocks

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
adjusting communities

Javi-er committed:
Adjusting map markers and adding the block to install

Javi-er committed:
Update default.png

michael-wynne-wsu commented:
Thanks @Javi-er 

A few small adjustments to make...
Changing some of the default text:
- Mukurtu v4 Featured Content > Featured Content
- Mukurtu Browse by Map > Browse by Map
- Mukurtu v4 Hero Image and Site Description > Welcome to Your Mukurtu CMS Site
- Welcome to our Mukurtu digital heritage site. > To start using your Mukurtu site, create a community, cultural protocol, and category.
- Featured content will appear here. > [nothing]

Minor appearance notes:
- There is a bit of extra space between the map title and the map compared to the other blocks.
- Don't need the horizontal divider below the final block of the page.
- For browse by community, Nick made this change on the communities page, it needs to apply here as well: https://github.com/MukurtuCMS/Mukurtu-CMS/pull/1025 

And then some bigger things:
- Featured content block needs to be styled.
- One-column hero block needs to be added (unless I missed that somewhere).

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
Merge branch 'main' into 1008--landing-page-and-blocks

Javi-er committed:
adding vertical hero block, adjusting styling and labels

Javi-er committed:
Update core.entity_view_display.node.landing_page.default.yml

Javi-er committed:
theming image with description vertical

Javi-er commented:
@michael-wynne-wsu I made those adjustments except for theming featured content, it keeps not working in my local.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
Adding full image with description block

Javi-er committed:
updating config

Javi-er committed:
adjusting styling

Javi-er committed:
adjusting styling

Javi-er committed:
Merge pull request #1046 from MukurtuCMS/1034--full-bleed-homepage-background

1034  Full-bleed homepage background custom block

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
updating media support

Javi-er committed:
Merge branch 'main' into 1008--landing-page-and-blocks

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
featured content structure

Javi-er committed:
Merge branch '1008--landing-page-and-blocks' of github.com:MukurtuCMS/Mukurtu-CMS into 1008--landing-page-and-blocks

Javi-er committed:
theming featured content

Javi-er committed:
adjusting config

Javi-er committed:
adjusting config

Javi-er committed:
adjusting basic page

Javi-er commented:
@michael-wynne-wsu I have implemented all the changes including theming featured content, let me know if you notice anything.

Javi-er committed:
adjusting remote video

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

michael-wynne-wsu commented:
@Javi-er Looking great, a few small notes if you have time to adjust!

1) Featured content
    - "view content" link should be styled like the browse by community names - Sentence case, fixed arrow glyph.
    - The content type labels should also be sentence case.
    - Any chance you could add a preview style for dictionary words? (not sure if it's that or that any featured media that is audio has no styling)
2) Vertical welcome
    - I think making the media full-width would be better
3) Map
    - It will wrap and show duplicate continents (documented here as well). Might need to make it taller to address that?
4) Browse by community
    - This is more vague, but the three column looks so empty compared to the other blocks. DO you think going to 4 column would fit better?

Let me know if you have questions!

<img width="1920" height="4502" alt="Mukurtu-Homepage-Mukurtu-CMS-08-14-2025_09_52_AM" src="https://github.com/user-attachments/assets/a6292630-e5f1-471f-ba0a-cb3866552a6b" />

Javi-er commented:
@michael-wynne-wsu the following is complete:

1) Featured content
    - [x]  "view content" link should be styled like the browse by community names - Sentence case, fixed arrow glyph.
    - [x] The content type labels should also be sentence case.
    - [x] Any chance you could add a preview style for dictionary words? (not sure if it's that or that any featured media that is audio has no styling)
2) Vertical welcome
    - [x] I think making the media full-width would be better
     It makes sense, I did that already for videos.
3) Map
    - [ ] It will wrap and show duplicate continents. Might need to make it taller to address that?
    - https://github.com/MukurtuCMS/Mukurtu-CMS/issues/724 for reference
    It should be possible to set up bounding box coordinates for preventing this to happening maybe, I'm familiar with MapBox and GoogleMaps, but not with Leaflet maps, I'll need to research it.
4) Browse by community
    - [ ] This is more vague, but the three column looks so empty compared to the other blocks. DO you think going to 4 column would fit better?
    I think that four will be more in line with the other blocks yes.

Javi-er committed:
adjusting feedback

michael-wynne-wsu commented:
@Javi-er 

Looking good! These are tiny things, but...

- [ ] Just so I'm aware, what is the target dimensions/ratio for the vertical welcome image? (I'll want to include dimensions for all of our themed elements for users in documentation at some point)
- [ ] Could the "View content" label layer on top of the preview, same as the community labels? (the rest of that looks good)
- [ ] Remove the margin under the featured content preview, just to bring the text below up a bit closer, I think.
- [ ] Yes, please switch community browse to 4-column.

I wouldn't worry about the map unless you just think it's a good way to use what little time is left. Alternatively, fixing the devel menu/nav menu z-index would be nice!

Javi-er committed:
adjusting feedback

Javi-er commented:
@michael-wynne-wsu 

1-  The hero image for image with description vertical and full are 1440 x 600 at desktop size.
2- " Could the "View content" label layer on top of the preview, same as the community labels? (the rest of that looks good)" Do you want to replace "view content" with the actual content label as it is on communities? the title field is quite long, so that button can end up covering the media, but I can do that, please confirm it.
3- I removed the margin below the media.
4- I switched communities to four columns now.

michael-wynne-wsu commented:
> @michael-wynne-wsu
> 
> 1- The hero image for image with description vertical and full are 1440 x 600 at desktop size. 2- " Could the "View content" label layer on top of the preview, same as the community labels? (the rest of that looks good)" Do you want to replace "view content" with the actual content label as it is on communities? the title field is quite long, so that button can end up covering the media, but I can do that, please confirm it. 3- I removed the margin below the media. 4- I switched communities to four columns now.

Re 2) No, keep "view content", just place the label lower

<img width="1856" height="539" alt="Screenshot 2025-08-14 at 2 24 30 PM" src="https://github.com/user-attachments/assets/91b2d846-fa60-4d2e-aa79-be4c79bf0965" />

Re 4) I was thinking of the four column community block matching the width of the four column category block.


Everything else is perfect!

Javi-er committed:
adjusting ink position

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

michael-wynne-wsu commented:
@Javi-er view content is perfect!

Community previews should be wider (same proportions as category?) to fill out the block width, if that makes sense...

<img width="1814" height="891" alt="Screenshot 2025-08-14 at 2 50 15 PM" src="https://github.com/user-attachments/assets/596fbb22-ca26-425b-a7c7-164ec683baa1" />

Javi-er committed:
adjusting margins

Javi-er commented:
@michael-wynne-wsu sorry, I'm using my laptop and there weren't padding differences at this screen size. I just adjusted it so it matches, I also gave some padding to the featured content block since it was touching the edges, let me know if now it's better.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

Javi-er committed:
adjusting grid

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr1032-3230fcgxgxiygsghy9abrv4biszmdegg.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68924722735236489f1dc40c

michael-wynne-wsu commented:
@Javi-er perfect! I think this can be merged in.
